gradle/published-java-module.gradle

```diff
@@ -99,9 +99,21 @@ publishing {
 signing {
 	var signingKey = resolveSigningKey()
 	var signingPassword = findSigningProperty( "signingPassword" )
-	useInMemoryPgpKeys( signingKey, signingPassword )
 
-	sign publishing.publications.publishedArtifacts
+	gradle.taskGraph.whenReady {tg ->
+		if (tg.hasTask(project.tasks.releaseChecks)) {
```

Comment:
What if I try to execute signing by itself?  This is not the proper task to check for - they only happen when doing a release.  If we re-configure signing, it would be nice to test signing without doing a release right?